src/main/scala/wdl4s/WdlNamespace.scala

```diff
@@ -525,6 +525,21 @@ object WdlNamespace {
             Option(new SyntaxError(wdlSyntaxErrorFormatter.undefinedMemberAccess(memberAccessAst)))
         }
       }
+
+      val identifiers = ast.getAttribute("value").findSingleIdentifiers()
+      // Validates single identifiers. They can be declarations but not calls or workflows
+      val identifierValidation = identifiers flatMap { singleIdentifier =>
+          val resolvedScope = call.resolveVariable(singleIdentifier.sourceString)
+        resolvedScope match {
+          case Some(_: Call) =>
+            Option(new SyntaxError(wdlSyntaxErrorFormatter.illegalIdentifier(singleIdentifier)))
+          case None =>
```

Comment:
No it's if we can't find any scope that corresponds to this identifier.
The first cases encompasses both TaskCall and WorkflowCall